src/mmw/mmw/settings/base.py

```diff
@@ -379,6 +379,7 @@ def get_env_setting(setting):
 
 # Geoprocessing Settings
 GEOP = {
+    'cache': bool(environ.get('MMW_GEOPROCESSING_CACHE', True)),
```

Comment:
`MMW_GEOPROCESSING_CACHE` comes in as a string and `bool('0') == True`. We'll need to cast to an `int` or something, or base this check on `MMW_GEOPROCESSING_CACHE` having some value.